sw/cheri/common/sonata-devices.hh

```diff
@@ -13,11 +13,13 @@
 #include <cheri.hh>
 #include <platform-gpio.hh>
 #include <platform-uart.hh>
+#include "../common/usbdev-utils.hh"
```

Comment:
I think the file (`sonata_devices.hh`) should be including only platform-usbdev.hh; common/usbdev-utils.hh is higher-level code than the RTOS-level driver itself.
`usbdev_tests.hh` should then include `../common/usbdev-utils.hh` to gain access to UsbdevUtils.